packages/api/lambdas/sqs-message-consumer.js

```diff
@@ -61,23 +63,26 @@ async function processQueues(event, dispatchFn) {
       visibilityTimeout,
       deleteProcessedMessage: false,
     });
-    log.info(`processing queue ${queueUrl}`);
 
+    log.info(`Processing queue ${queueUrl}`);
     const messageConsumerFn = dispatchFn.bind({ rulesForQueue });
+
     return consumer.consume(messageConsumerFn);
   }));
 }
 
 /**
- * Process an SQS message
+ * Archive and process an SQS message
  *
  * @param {string} queueUrl - Queue URL for incoming message
  * @param {Object} message - incoming queue message
  * @returns {Promise} - promise resolved when the message is dispatched
  */
-function dispatch(queueUrl, message) {
+async function dispatch(queueUrl, message) {
   const messageReceiveCount = Number.parseInt(message.Attributes.ApproximateReceiveCount, 10);
   const rulesForQueue = this.rulesForQueue;
+  log.info(`Archiving messages from queue ${queueUrl}`);
```

Comment:
I would remove this log from here and put it in `archiveSqsMessageToS3`, if anything. Because the log is specific to the logic in that function itself

Reply:
Makes sense to me